dtfs-central-api/src/v1/controllers/utilisation-report-service/upload-utilisation-report.controller.js

```diff
@@ -32,7 +32,13 @@ const postUtilisationReportData = async (req, res) => {
       return res.status(400).send(validationErrors);
     }
 
-    // TODO: FN-967 want to 429 here if a report details already exists
+    // If a report already exists for this month/year/bank combo, return 409
```

Comment:
This comment feels a bit unnecessary as the intention seems clear enough from the code. I think `existingReports` should be singular though.